src/Collection/utils.ts

```diff
@@ -2,26 +2,56 @@ import { utils } from 'decentraland-commons'
 import { getCurrentNetworkURNProtocol } from '../ethereum/utils'
 import { CollectionAttributes, FullCollection } from './Collection.types'
 
+const tpwCollectionURNRegex = /^urn:decentraland:([^:]+):ext-thirdparty:([^:]+)$/
+
 export function getDecentralandCollectionURN(
   collectionAddress: string
 ): string {
   return `urn:decentraland:${getCurrentNetworkURNProtocol()}:collections-v2:${collectionAddress}`
 }
 
+export function getThirdPartyCollectionURN(urn_suffix: string) {
+  return `urn:decentraland:${getCurrentNetworkURNProtocol()}:ext-thirdparty:${urn_suffix}`
+}
+
 export function toFullCollection(
   dbCollection: CollectionAttributes
 ): FullCollection {
   return {
     ...utils.omit(dbCollection, ['urn_suffix']),
-    urn: getDecentralandCollectionURN(dbCollection.contract_address),
+    urn: dbCollection.urn_suffix
+      ? getThirdPartyCollectionURN(dbCollection.urn_suffix)
+      : getDecentralandCollectionURN(dbCollection.contract_address),
   }
 }
 
 export function toDBCollection(
   collection: FullCollection
 ): CollectionAttributes {
+  const isTPW = isTPCollection(collection.urn)
+  let urn_suffix = isTPW ? decodeTPCollectionURN(collection.urn)[1] : null
+  let eth_address = isTPW ? '' : collection.eth_address
+  let contract_address = isTPW ? '' : collection.contract_address
+  let salt = isTPW ? '' : collection.salt
+
   return {
-    ...utils.omit(collection, ['urn']),
-    urn_suffix: null,
+    ...utils.omit(collection, ['urn', 'lock']),
+    urn_suffix,
+    eth_address,
+    contract_address,
+    salt,
+  }
+}
+
+export function isTPCollection(urn: string) {
+  return tpwCollectionURNRegex.test(urn)
+}
+
+export function decodeTPCollectionURN(urn: string): [string, string] {
```

Comment:
I think it'd be useful to return an object with names for each value. I found `decodeTPCollectionURN` in the code above and couldn't really tell what the response value was. Reading the implementation I can't really tell either

What do you think?